core/math/a_star_grid_2d.cpp

```diff
@@ -579,7 +620,7 @@ TypedArray<Dictionary> AStarGrid2D::get_point_data_in_region(const Rect2i &p_reg
 
 			Dictionary dict;
 			dict["id"] = p.id;
-			dict["position"] = p.pos;
+			dict["position"] = _get_solid_unchecked(p.id);
```

Comment:
While fixing the git history in this PR to remove the merge commit and duplicated change history, I noticed this was the only difference between the older version (db18fe5ac0e0e23446abf1d66012dce2b1e15f18) and your rebased version from yesterday.

Please confirm whether this extra change was indeed intentional.

Reply:
Yes this was intentional